content/docs/use-cases/experiment-tracking.md

```diff
@@ -0,0 +1,101 @@
+```
+
+![]() _Organized experiments in DVC_
+
+> 📖 Ready to dive in? See [Get Started: Experiments](/doc/start/experiments).
+
+Unlike tools that focus on experiment navigation, DVC's approach guarantees
+reproducibility by working on top of Git, instead of on being a side system with
+fragile links to Git. This means that your [data is
+tracked][versioning everything] from within project versions (without storing it
+in Git). It also enables distributed collaboration along the way, via optional
+hosting like GitHub or GitLab.
+
+DVC projects provide a seamless, local-first experience (no need for running
+servers, special services, or loading websites). You control where and how your
+data is saved and shared. DVC also improves storage efficiency via
+<abbr>caching</abbr>, preventing repetitive data transfers on every experiment.
+
+On top of all that, DVC is completely language agnostic. You can expect the same
+convenience and performance whether you're using Jupyter Notebooks or Scala, CSV
+data frames or HDFS.
+
+|                       | DVC Experiments                         | MLFlow                                         | Weights & Biases          | TensorBoard             |
+| --------------------- | --------------------------------------- | ---------------------------------------------- | ------------------------- | ----------------------- |
+| Experiment definition | `dvc.yaml` metafiles                    | Implicit (in code)                             | Implicit                  | Implicit                |
+| How to run it         | `dvc exp run` any terminal command      | Code integration                               | Code (Python only)        | Code (Python)           |
+| Data storage          | [User][storage] (yours e.g. cloud)      | User (yours)                                   | Hosted (theirs)           | Limited hosted (theirs) |
+| Compute               | User (e.g [CI/CD][cml])                 | User or [hosted][dbricks]                      | User (e.g. local)         | User (local)            |
+| What can be tracked   | code & data, params, metrics, artifacts | data, params, metrics                          | data, params, environment | params and metrics      |
+| Visualization         | Terminal + [web][dvc studio]            | Web app                                        | Website                   | Website                 |
+| How it's captured     | Custom Git commit                       | Database                                       | Database                  | Metadata files          |
+| What you pay          | Your resources + optional [web UI]      | Your resources + [hosting][dbricks] (optional) | [Subscription][wandb]     | Your resources          |
+
+[storage]: /doc/command-reference/remote/add#supported-storage-types
+[dbricks]: https://databricks.com/product/pricing
+[web ui]: https://studio.iterative.ai/#pricing
+[wandb]: https://wandb.ai/site/pricing
```

Comment:
New table approach @shcheklein . Looks like this rn:

![image](https://user-images.githubusercontent.com/1477535/141966065-70d75f2b-15d3-466a-8cd9-6d161f610b6d.png)

Reply:
Table thoughts:

* Looks like a lot of rows, and also a lot of text, which might be useful as a reference but makes it hard to digest at a glance. The question idea is nice because it's less technical, but it's sort of inconsistent, and it's more wordy.
* I don't see it really highlighting the advantages of DVC, like automated reproducibility, data and pipeline codification and versioning, local/serverless/distributed tracking and sharing, and connection to workspace and git repo (like being able to switch experiments in your workspace).
* Are we worried about accuracy for now or should we leave it until we are set on the table format?

Reply:
- Agree. It's hard to keep it lean ⌛ 
- We are trying to mention those other features in the use case story (text) since we can't fit all that into a table. The table focuses on experiment tracking directly (on comparable criteria). Did the [previous table](https://github.com/iterative/dvc.org/pull/2782#pullrequestreview-807086370) cover them?
- Table format first but at this point specific corrections are also welcome (the actual values are meant to be correct now)

I'll ping you soon, working on another iteration of the table based on more feedback I got privately... ⌛

Reply:
Another iteration on this table approach. It's getting long...

![image](https://user-images.githubusercontent.com/1477535/144355963-83cf2d0c-775c-4af9-a81c-6aa27b13f794.png)